tests/test_content.py

```diff
@@ -8,6 +8,7 @@
 
 
 @pytest.mark.nondestructive
+@pytest.mark.slow
```

Comment:
Flip the order on these. It's a bit weird but decorators are executed in order from the bottom up. We want `@pytest.nondestructive` to be evaluated first.